config-user.default.php

```diff
@@ -53,13 +53,13 @@
 	'sort_order' => 'DESC',
 	'anon_access' => false,
 	'mark_when' => array (
-		'article' => true,
+		'article' => false,
 		'gone' => false,
 		'max_n_unread' => false,
 		'reception' => false,
 		'same_title_in_feed' => false,
-		'scroll' => true,
-		'site' => true,
+		'scroll' => false,
+		'site' => false,
```

Comment:
" while scrolling "
"when the article is opened on its original website "

Reply:
first: I like
second: I do not like this change

Reply:
Having the second as true may be worse than the outside of text area setting. You know you're clicking on it, but you may not even realize you're accidentally marking things as read in the background.

But you could make an argument that it's similar to opening the article within the FreshRSS UI.

Reply:
No strong opinion on this one but I join math, I think it makes more sense to mark it as read when you open the article on the original website.

Reply:
fixed